src/features/game/events/minigames/claimMinigamePrize.test.ts

```diff
@@ -215,4 +226,121 @@ describe("minigame.prizeClaimed", () => {
       ].prizeClaimedAt
     ).toEqual(date.getTime());
   });
+
+  it("claims a faction marks prize", () => {
+    const date = new Date("2024-05-05T00:00:00");
```

Comment:
Might want to make sure all timestamps in all the affected files have the trailing `Z`.